### ISSUE TITLE
Problem with dashes in the key names

### ISSUE THREAD

**Issue by labradoruk** [stevearc/dql#12]
When trying to query a dynamo table which has a primary index called 'foo-bar' :
```
select * from my-prod-table where foo-bar = 'something';
```

it fails with the error: 
```
ValidationException: Invalid KeyConditionExpression: Syntax error; token: "-", near: "foo-bar"
```

Any available workaround? 

Thanks

**Comment by stevearc** [comment]
Yyyyyyyep looks like that was a bug. Basically, I couldn't find it documented anywhere but Dynamo doesn't allow fields with dashes in the [KeyConditionExpression](http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html#DDB-Query-request-KeyConditionExpression), so now we'll detect when a field has a dash and replace it with a value in [ExpressionAttributeNames](http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html#DDB-Query-request-ExpressionAttributeNames)

Released the fix in version [0.5.24](https://pypi.python.org/pypi/dql/0.5.24)

**Comment by aterreno** [comment]
Great thanks for the quick fix!
